.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+ - [ ] Ready
+ - [ ] In development
+ - [ ] Hold
+
+## Closes issue(s)
+A list of issues that are closed by this PR
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# Changes include
+- [ ] Bugfix (non-breaking change that solves an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
+- [ ] Refactor (non-breaking/breaking change that refactor the previous code base)
+
+## Impacted Areas in Application
+List general components of the application that this PR will affect:
+
+## Todos
+- [ ] Updated and Passed Tests
+- [ ] Passed Quality Checks
+- [ ] Updated  Documentation
+
```

Comment:
Maybe we can add a field to schedule a meeting with the team to see if the PR is impacting other areas of the code.

Reply:
Is there a way to do that using links?

Reply:
Not sure what you mean by links.

Reply:
Lets keep it status quo, and we can research on it later. I don't think there is a field for scheduling. Unless you have any suggestions....